### ISSUE TITLE
how to get `Ptr<MotionEstimatorBase>` by `MotionEstimatorRansacL2`

### ISSUE THREAD

**Issue by chanble** [twistedfall/opencv-rust#217]
code like this:
```
    let est_para = 0.1f32;
    let mut est = MotionEstimatorRansacL2::new(MotionModel::MM_AFFINE).unwrap();
    let ransac = RansacParams::new(
        3, 5.0, 0.5, 0.8
    ).unwrap();

    est.set_ransac_params(&ransac);
    est.set_min_inlier_ratio(est_para);

    // est need convert to Ptr<dyn MotionEstimatorBase> type ,  Ptr::<dyn MotionEstimatorBase>::new(est) is not work
   // I am new ruster, please help me
    let mut motion_est_builder = KeypointBasedMotionEstimator::new(est).unwrap();
```

**Comment by twistedfall** [comment]
Hi, it looks like it's not really possible at the moment. I will try to fix it as soon as possible, thanks for the report!

**Comment by twistedfall** [comment]
Can you please check the `rel` branch? You should now be able to create smart pointer with just:
```rust
let est_ptr = Ptr::new(est);
```
and then pass it to `KeypointBasedMotionEstimator::new` with:
```rust
let mut motion_est_builder = KeypointBasedMotionEstimator::new(est_ptr.into()).unwrap();
```

**Comment by twistedfall** [comment]
I have also released the 0.50.0 version, which contains this fix, feel free to reopen the issue if the problem is still there.

**Comment by chanble** [comment]
thank you! It works
